docs/standard/getting-started.md

```diff
@@ -26,7 +26,7 @@ There are a number of ways to get started with .NET.  Because .NET is a massive
 
 ## Getting started using .NET core
 
-* [Getting Started with .NET Core](../core/getting-started.md) provides an overview of articles which show how to get started with .NET Core on different operating systems and using different tools.
```

Comment:
should you also add the installation topic(s) here in this article as a separate section?